site/src/components/LicenseBanner/LicenseBanner.tsx

```diff
@@ -1,19 +1,9 @@
-import { useActor } from "@xstate/react"
-import { useContext, useEffect } from "react"
-import { XServiceContext } from "xServices/StateContext"
+import { useDashboard } from "components/Dashboard/DashboardProvider"
 import { LicenseBannerView } from "./LicenseBannerView"
 
 export const LicenseBanner: React.FC = () => {
```

Comment:
👀 ug why isn't the lint rule picking up this `React.FC` import? Something to look into later.

Reply:
Yeah, I usually use what auto import uses but I will change it.

Reply:
Oh, you don't have to change it - this is more of a note for myself. I'm sure there are other instances in the repo.